### PR TITLE
s4553987_Segment the ISICs data set with the Improved UNet

### PR DESCRIPTION
**Pull request by w895540347**

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Commit messages should be more specific - what exactly was changed
- Code valid for Normal Difficulty
- No quantitative result 
- No plots, though you did include functions to make plots.  Where did they go? It is hard to the your results

Other Feedback
- Should provide at least some description in your PR

**Comment by rexsw** [comment]
Code looks good for normal difficulty

For next time;

Your pull request should have a description of what it is and why it should be merged 

Comments such as 
    # input layer
     inputs = Input((img_size, img_size, 3))
Add nothing to the readability of the code and should be avoided. Overall comments were well done

Code in files like driver scripts should be in a main function.

File: recognition/s4553987 Segment the ISICs data set with the Improved UNet/main.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import os
```
Comment (shakes76) [inline recognition/s4553987 Segment the ISICs data set with the Improved UNet/main.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Data usage, Readme OK
Module present
No plots or quantitative results
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info. 
Avoid using spaces in folder/file names

**Pull request merged**